### ISSUE TITLE
AUC-ROC

### ISSUE THREAD

**Issue by Ostrich5yw** [yunseokddi/DeepSAD_pytorch#1]
Hello, When I run this code on my dataset, I often find that AUC-ROC is less than AUC-PR.
Is there a problem with my data set or is this a normal situation?
I would appreciate your reply :)

**Comment by yunseokddi** [comment]
I think this problem is case by case.
What's your test data's ratio? (Normal : Abnormal)
If your test data set is unbalance, your AUC-ROC is less than AUC-PR.

**Comment by Ostrich5yw** [comment]
Since I'm doing industrial fault detection, this requires very little abnormal data. The proportion of abnormalities was 2% in the training set and 10% in the test set.

**Comment by yunseokddi** [comment]
I think you should use "Self-supervised learning based model" than "Semi-supervised learning based model". Because your abnormal dataset's size is very low. So If you use semi-supervised learning based model, you get the low AUC-ROC or PR.
Are you use image preprocessing? For example cropping interested area, image correction and etc...
